code.swift

```diff
@@ -57,7 +57,7 @@ class PeriodicTableViewController: UIViewController {
     @IBAction func chooseRandomElement(_ sender: Any) { // Space needed before opening bracket
         let randomElement = elementsContoller.elements.randomElement() // Better variable name
         
-        var chosenElementView: ElementView!
+        var chosenElementView: ElementView? // Why the force unwrapped?
```

Comment:
Because we are “sure” that we’ll find an entry, so no need to have a regular optional here. If you change this to a regular optional, you’ll also need to check for it when we highlight that element view below!